game-core/src/main/java/games/strategy/triplea/ui/AbstractStatPanel.java

```diff
@@ -39,13 +41,17 @@
    * @return all the alliances with more than one player.
    */
   public Collection<String> getAlliances() {
-    final Collection<String> alliances = new TreeSet<>();
+    return getAllianceMap().keySet();
+  }
+
+  public Map<String, Set<PlayerID>> getAllianceMap() {
```

Comment:
can this be protected? 

Also have you considered using guava multimap? I found they are funky to construct, but beyond that it is a data structure that is exactly meant for `Map<T, Collection<T>>`: http://www.baeldung.com/guava-multimap

Reply:
Updated visibility. Not really that big of a fan of multimap.